Example projects (JS / TS) for node backend use of MF

arimus opened the pull request:
This commit demonstrates use of Module Federation in node using a project that is using vanilla JS and one that is using TypeScript.

arimus committed:
updated js and ts project examples for node dynamic remote loading

arimus committed:
remove old dynamic-remote-nodes project and replace with the new one

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/module-federation/module-federation-examples?pullRequest=4316) <br/>All committers have signed the CLA.

arimus committed:
fix for project dynamic-remotes-node-javascript rename to dynamic-remotes-node

arimus commented:
I fixed the dynamic-remotes-node-javascript to dynamic-remotes-node that was in the original pull request :P And fixed the email address associated for the CLA signing to match up
